bin/hipcc

```diff
@@ -333,7 +332,11 @@ if ($HIP_PLATFORM eq "clang") {
 }
 
 # Add paths to common HIP includes:
-$HIPCXXFLAGS .= " -isystem $HIP_INCLUDE_PATH" ;
+if ($HIP_PLATFORM eq "clang") {
+    $HIPCXXFLAGS .= " -I $HIP_INCLUDE_PATH" ;
```

Comment:
this may result in lots of warning msg for each HIP compilation. Should we clean that up before making this change?

Reply:
Could elaborate in details?

Reply:
-isystem suppresses warnings in these headers. Do you see any extra warnings from hip_runtime.h when making this change?

Reply:
So far, from the HIP test compilation results, that's only warning I observed.

Reply:
Hi @yxsamliu 
Does the new change look OK?